design/ceph/object/ceph-multisite-overview.md

```diff
@@ -48,6 +48,9 @@ To enable Ceph's multisite, the following steps need to happen.
 3. A master zone in the master zone group needs to be created
 4. An object store needs to be added to the master zone
 
+### Master Zone/Zonegroup
+A multi-site configuration requires a master zone group and a master zone. Additionally, each zone group requires a master zone. Zone groups may have one or more secondary or non-master zones. Even though data is replicated among different rgw servers, the master copy of realm's metadata such users, quota is stored in master zone in master zonegroup.
```

Comment:
I don't understand about the users and quota. Could you rephrase? This is saying users metadata **and** quota is stored in the master zone?

Reply:
@travisn : Right. If that kind of request send it will be fetched or redirected to the master zone.AFAIR data request can be handled by all the rgw servers, but metadata request will send to rgw server master zone.
@cbodley : Am I missing something??

Reply:
you're right @thotz, i just think the wording is a bit confusing. maybe something like this?

> The master zone of the master zonegroup is designated as the 'metadata master zone', and all changes to user and bucket metadata are written through that zone first before replicating to other zones via metadata sync. This is different from data sync, where objects can be written to any zone and replicated to its peers in the zonegroup.